core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -57,6 +63,33 @@ public static void initPageVariables(JellyContext context) {
         context.setVariable("icons", icons);
     }
 
+    public String getIonicon(String name, String title) {
+        if (ionicons.containsKey(name)) {
+            return ionicons.get(name);
+        }
+
+        // Load icon if it exists
+        try {
+            String ionicon = new String(
+                    Files.readAllBytes(Paths.get("war/src/main/webapp/images/ionicons/" + name + ".svg")));
+
+            ionicon = ionicon.replaceAll("(<title>)[^&]*(</title>)", "$1$2");
+            ionicon = ionicon.replaceAll("<svg", "<svg aria-hidden=\"true\"");
+            ionicon = ionicon.replace("stroke:#000", "stroke:currentColor");
+
+            ionicons.put(name, ionicon);
+
+            if (StringUtils.isNotBlank(title)) {
+                return "<span class=\"jenkins-visually-hidden\">" + title + "</span>" + ionicon;
+            }
+
+            return ionicon;
+        } catch (IOException e) {
```

Comment:
So this fixes the tests (loading from classpath) but then breaks it from `jetty:run`:

```
            InputStream inputStream = getClass().getResourceAsStream("/images/ionicons/" + name + ".svg");
            if (inputStream == null) {
                return getIonicon("ellipse-outline", "Placeholder");
            }

            String ionicon = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
```

This will only work from core as well, needs a look to see more how the rest of the Icon class fits together